libcloud/storage/drivers/azure_blobs.py

```diff
@@ -299,8 +323,9 @@ def _xml_to_object(self, container, blob):
             extra['md5_hash'] = value
 
         meta_data = {}
-        for meta in list(metadata):
-            meta_data[meta.tag] = meta.text
+        if metadata:
+            for meta in list(metadata):
+                meta_data[meta.tag] = meta.text
```

Comment:
same as above, is meta.tag guaranteed to be not None?

Reply:
Leaving as-is since it's not a required part of this fix.